lib/compat/wordpress-6.1/class-wp-theme-json-data.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * API to update a theme.json structure.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Class to update with a theme.json structure.
+ */
+class WP_Theme_JSON_Data {
```

Comment:
Apologies for commenting just after this PR merged, but should this line have a `Gutenberg` suffix, or have a `class_exists` check before defining it, so that we don't run into any issues when this is backported to core?

Reply:
Yes, it'd be the best to call `class_exists` before requiring this file.

Reply:
+1